src/collections/_documentation/workflow/integrations/integration-platform/index.md

```diff
@@ -3,166 +3,573 @@ title: Integration Platform
+However, if your integration has a Redirect URL configured, the integration redirects the user’s browser to the configured URL with the grant code and installation ID in the query params.
 
-Start your build by implementing the Redirect URL endpoint, `/setup` — typically where you exchange the grant code for an access token that allows you to make API calls to Sentry. 
+Start your build by implementing the Redirect URL endpoint, /setup — typically where you exchange the grant code for an access token that allows you to make API calls to Sentry.
 
 ```python
-import requests
-from flask import redirect, request
+    import requests
+    from flask import redirect, request
 
-@app.route('/setup', methods=['GET'])
-def setup():
-    code = request.args.get('code')
-    install_id = request.args.get('installationId')
-    
-    url = u'http://sentry.io/api/0/sentry-app-installations/{}/authorizations/'
+    @app.route('/setup', methods=['GET'])
+    def setup():
+        code = request.args.get('code')
+        install_id = request.args.get('installationId')
+        
+        url = u'https://sentry.io/api/0/sentry-app-installations/{}/authorizations/'
+        url = url.format(install_id)
+        
+        payload = {
+            'grant_type': 'authorization_code',
+            'code': code,
+            'client_id': 'your-client-id',
+            'client_secret': 'your-client-secret',
+        }
+        
+        resp = requests.post(url, json=payload)
+        data = resp.json()
+        
+        access_token = data['token']
+        refresh_token = data['refreshToken']
+        # ... Securely store the install_id, access_token and refresh_token in DB ...
+        
+        return redirect('http://sentry.io/settings/')
+```
+
+#### Verifying Installations (optional)
+
+Typically if you have the Redirect URL configured, there is work happening on your end to 'finalize' the installation. If this is the case, we recommend enabling the Verify Install option for your integration. Once enabled, you will need to send a request marking the installation as officially 'installed.'
+
+```python
+    requests.put(
+        'https://sentry.io/api/0/sentry-app-installations/{}/'.format(install_id),
+        json={'status': 'installed'},
+    )
+```
+
+In the case a user has not completed the setup on your end, whatever the case may be, Sentry will show the installation as 'pending' in the UI. 
+
+[ SCREENSHOT: screenshot of pending state ]
+
+#### Refreshing Tokens
+
+The Access Tokens you receive from Sentry expire after eight hours. To retrieve a new token, you’ll make a request to the same Authorization endpoint used in the /setup endpoint above, but with a slightly different request body.
+
+```python
+    def refresh_token(install_id):
+        url = u'http://sentry.io/api/0/sentry-app-installations/{}/authorizations/'
```

Comment:
http -> https
`url = u'https://sentry.io/api/0/sentry-app-installations/{}/authorizations/'`

Reply:
Do we need to specify the fact the string is unicode? Would't it work with:
```
'https://sentry.io/api/0/sentry-app-installations/{}/authorizations/'
```

Reply:
@MeredithAnya @scefali What's the final decision on this? :)

Reply:
@MimiDumpling we can take it out